api/v1beta1/consts.go

```diff
@@ -128,6 +128,8 @@ const (
 	DisksReadyCondition clusterv1.ConditionType = "DisksReady"
 	// NetworkInterfaceReadyCondition means the network interfaces exist and are ready to be used.
 	NetworkInterfaceReadyCondition clusterv1.ConditionType = "NetworkInterfacesReady"
+	// PrivateEndpointReadyCondition means the private endpoints exist and are ready to be used.
```

Comment:
Is this supposed to be singular or plural?

Reply:
I'm not sure, based on the above consts, they seem to be mixed. I can put it to be plural, don't have a preference